client/state/data-layer/wpcom/plans-request.js

```diff
@@ -0,0 +1,27 @@
+import wpcom from 'lib/wp';
```

Comment:
>I imagine this layer as the translation from the internal Calypso representation of data to the server's representation of data.

I really like this idea here. This could be good for our need to start being more responsible with the state we store, as we currently encounter storage quota limits when we're not using much of the data we're storing. The primary issue with transforming data into reducer state is needing to transform it once more when sending back to the server, but this abstraction layer could be a nice pattern for handling that.